<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="label_app_version">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_qt_icon">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_qt_version">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_app_icon">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_notifications">
         <property name="title">
          <string>Notifications</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QLabel" name="label_notification_duration">
            <property name="text">
             <string>Notification duration:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spin_priority">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_notification_duration_ms">
            <property name="text">
             <string>ms</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QCheckBox" name="cb_tray_icon_unread">
            <property name="text">
             <string>Change the tray icon color when there are unread notifications</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spin_duration">
            <property name="minimum">
             <number>500</number>
            </property>
            <property name="maximum">
             <number>30000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_notification_priority">
            <property name="text">
             <string>Minimum priority to show notifications:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QCheckBox" name="cb_notify">
            <property name="text">
             <string>Show a notification for missed messages after reconnecting</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_server_info">
         <property name="title">
          <string>Server info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="pb_change_server_info">
            <property name="text">
             <string>Change server info</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_advanced">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_watchdog">
         <property name="title">
          <string>Server watchdog thread (re&amp;quires restart)</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_watchdog_interval">
              <property name="text">
               <string>Interval:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spin_watchdog_interval">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_watchdog_interval_s">
              <property name="text">
               <string>s</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_cache">
         <property name="title">
          <string>Cache</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pb_clear_cache">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_open_cache_dir">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_cache">
            <property name="text">
             <string>0 MB</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_logging">
         <property name="title">
          <string>Logging</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <widget class="QComboBox" name="combo_logging"/>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>190</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pb_open_log">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Open logfile</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_logging">
            <property name="text">
             <string>Level</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_reset">
         <property name="text">
          <string>Reset all settings</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>spin_priority</tabstop>
  <tabstop>spin_duration</tabstop>
  <tabstop>cb_notify</tabstop>
  <tabstop>cb_tray_icon_unread</tabstop>
  <tabstop>pb_change_server_info</tabstop>
  <tabstop>groupBox_watchdog</tabstop>
  <tabstop>spin_watchdog_interval</tabstop>
  <tabstop>pb_clear_cache</tabstop>
  <tabstop>pb_open_cache_dir</tabstop>
  <tabstop>combo_logging</tabstop>
  <tabstop>pb_open_log</tabstop>
  <tabstop>pb_reset</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_clear_cache</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>clearCache()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_open_cache_dir</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>openCache()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_priority</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_duration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_notify</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_tray_icon_unread</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox_watchdog</sender>
   <signal>toggled(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_watchdog_interval</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_change_server_info</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>serverInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_reset</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>resetSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clearCache()</slot>
  <slot>openCache()</slot>
  <slot>saveSettings()</slot>
  <slot>changed()</slot>
  <slot>serverInfo()</slot>
  <slot>titleFont()</slot>
  <slot>dateFont()</slot>
  <slot>messageFont()</slot>
  <slot>resetFontsSizes()</slot>
  <slot>resetSettings()</slot>
  <slot>applicationFont()</slot>
  <slot>selectedApplicationFont()</slot>
  <slot>applicationIcon()</slot>
  <slot>messageApplicationIcon()</slot>
  <slot>statusLabel()</slot>
  <slot>mainButton()</slot>
  <slot>messageButton()</slot>
  <slot>showPriorityChanged()</slot>
  <slot>messagePriority()</slot>
 </slots>
</ui>
